app/views/users/index.html.haml

```diff
@@ -0,0 +1,32 @@
+= content_for :page_title, 'RDA Team'
```

Comment:
We should look at throwing all the titles into locale at some stage similar to what you've done with devise
http://guides.rubyonrails.org/i18n.html